WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -156,21 +173,26 @@ Deploying ReFS as a backup target is best suited for applications and hardware t
 | Hard links | Yes<sup>1</sup> | Yes |
 | Object IDs | No | Yes |
 | Offloaded Data Transfer (ODX) | No | Yes |
-| Short names | No | Yes |
+| Short names (SFN 8.3) | No | Yes |
 | Extended attributes | No | Yes |
 | Disk quotas | No | Yes |
 | Bootable | No | Yes |
 | Page file support | No | Yes |
 | Supported on removable media | No | Yes |
+1. This feature becomes available on **newly formatted ReFS 3.5 volumes only**
+Prerequisites: Windows 10 Enterprise / Windows 10 for Workstations or Windows Server vNext, starting from build 19536 or later.
+Disclaimer: Any in-place upgrade to this version will upgrade ReFS attached volumes to the latest version, but will unlock the use of hardlinks. 
```

Comment:
Hmm. I feel a bit uneasy with this phrasing and the use of the term "unlock" here. Looking at the how it currently reads, the meaning does not seem to be obvious, at least not to me. I would like to ask if you meant "break", "discontinue", "turn off", or "unlink", because the current phrasing looks like it means 'unlock' in the meaning of "unlock the next level" or "unlock a new feature" in the software. If you mean 'unlock' as in "turn off" or "break", I suggest using a term with a more direct and clear meaning, like for example:
```suggestion
Disclaimer: Any in-place upgrade to this version will upgrade ReFS attached volumes to the latest version, but will also break current hardlinks. 
```
My suggestion may very well be not precise enough, because I don't actually know exactly what happens to hardlinks when ReFS is upgraded from version 3.5 to a newer one.

Reply:
Depending on the correct understanding of the hardlink feature and upgrading ReFS to a newer version, it could also be phrased like this:
```suggestion
Disclaimer: Any in-place upgrade to this version will upgrade ReFS attached volumes to the latest version, but will disable existing usage of hardlinks. 
```

Reply:
Hi @illfated thanks for spotting this. It should be 

```suggestion
Disclaimer: Any in-place upgrade to this version will upgrade ReFS attached volumes to the latest ReFS version, but will **not** make the hardlinks feature available.
```